Tensile/Source/client/source/BenchmarkTimer.cpp

```diff
@@ -81,28 +82,22 @@ namespace Tensile
         {
             m_numEnqueuesInSolution = 0;
             m_timeInSolution = double_millis::zero();
-
+            
             ContractionSolution::ProjectedPerformance pp =
               solution.projectedPerformance(m_problem, m_hardware);
-
+            m_solution = solution;        
+    
             m_reporter->report(ResultKey::Tile0Granularity, pp.tile0Granularity);
             m_reporter->report(ResultKey::Tile1Granularity, pp.tile1Granularity);
             m_reporter->report(ResultKey::CuGranularity, pp.cuGranularity);
             m_reporter->report(ResultKey::WaveGranularity, pp.waveGranularity);
             m_reporter->report(ResultKey::TotalGranularity, pp.totalGranularity);
-
-            m_reporter->report(ResultKey::TilesPerCu, pp.tilesPerCu);
-
             m_reporter->report(ResultKey::TilesPerCu, pp.tilesPerCu);
-
-            m_reporter->report(ResultKey::AluUs, pp.staticModel.aluUs);
-            m_reporter->report(ResultKey::MemReadUs, pp.staticModel.memReadUs);
-            m_reporter->report(ResultKey::MemWriteUs, pp.staticModel.memWriteUs);
-
+            
             m_reporter->report(ResultKey::MemReadBytes, pp.staticModel.memReadBytes);
             m_reporter->report(ResultKey::MemWriteBytes, pp.staticModel.memWriteBytesD);
-
-            // TODO-perfcounter - add memory global reads and writes from performance counter
+            m_reporter->report(ResultKey::MemGlobalReads, pp.staticModel.memGlobalReads);
+            m_reporter->report(ResultKey::MemGlobalWrites, pp.staticModel.memGlobalWrites);
```

Comment:
the comment referred to a future feature where we use rocprof APIs to read memory bandwidth performance counters and compare those against the values predicted by the model.  I think these new counters are something different - not sure they are needed